src/main/groovy/com/palantir/gradle/docker/DockerExtension.groovy

```diff
@@ -85,12 +87,25 @@ class DockerExtension {
         copySpec.from(files)
     }
 
-    public Set<String> getTags() {
-        return tags
+    public Set<String> getUnresolvedTags() {
+        return unresolvedTags
     }
 
+    @Deprecated
     public void tags(String... args) {
-        this.tags = ImmutableSet.copyOf(args)
+        this.unresolvedTags.addAll(args)
```

Comment:
I don't think we need/want to change the behavior of this method. Specifically it should continue to overwrite any previously set tags.

Reply:
Ok, got it